### DISCUSSION TITLE
Small error in the docs, in the Handle JWT tokens section

### DISCUSSION THREAD

**Discussion by leynier** [fastapi/fastapi#7025] [Questions]
In [Handle JWT tokens](https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/#handle-jwt-tokens) tutorial section the variable [ACCESS_TOKEN_EXPIRE_MINUTES](https://github.com/tiangolo/fastapi/blob/b53c443a06f898189e0f458dd1d7cbe55742192b/docs_src/security/tutorial004.py#L14) never is used in [create_access_token function](https://github.com/tiangolo/fastapi/blob/b53c443a06f898189e0f458dd1d7cbe55742192b/docs_src/security/tutorial004.py#L78), is used [15 minutes](https://github.com/tiangolo/fastapi/blob/b53c443a06f898189e0f458dd1d7cbe55742192b/docs_src/security/tutorial004.py#L83).

**Comment by tiangolo** [comment]
Thanks for closing it. It seems this is working correctly, maybe it was recently fixed.
